src/haiway/context/state.py

```
@@ -56,6 +56,40 @@ def __delattr__(
             f" attribute - '{name}' cannot be deleted"
         )
 
+    def has_state[StateType: State](
```

Pylint (3.3.7)

[error] Parsing failed: 'expected '(' (haiway.context.state, line 59)'

(E0001)